frontend/src/components/Events/EventParticipants.js

```diff
@@ -27,7 +27,6 @@ function EventParticipants(props) {
       <Typography variant="h5">{"Participants (" + participants.length + ")"}</Typography>
       <List1>
         {_.isEmpty(participants) ? (
-          // <CircularProgress color="primary" size={100}/> // once we have LOADING state set up
```

Comment:
Can you just get replace line 29 if youre not returning anything of value in the true case? Like: _.isEmpty(participants) && {your component}

Reply:
Yea for sure for sure. Ez change. Of course I should do it. Its my mistake after all :))))))))